Vagrantfile

```diff
@@ -17,7 +17,7 @@ fi
 SCRIPT
 
 Vagrant.configure("2") do |config|
-  config.vm.box = "mpasternak/focal64-arm"
+  config.vm.box = "jeffnoxon/ubuntu-20.04-arm64"
```

Comment:
What do we use this VagrantFile for?

Reply:
To run php locally on my machine, we should remove this file though and put it in the gitignore. I'll draft a quick follow up pr.